install

```diff
@@ -95,6 +95,13 @@ do_install()
   # Homebrew python gets us bottles for numpy, scipy, etc.
   brew tap homebrew/python
 
+  # Sierra no longer has a working qt4 so use this private version until the issue is resolved.
+  if [[ `sw_vers -productVersion` > "10.11" ]]
+  then
+    brew install cartr/qt4/qt
+    SKIP_KEYS="--skip-keys=pyqt"
```

Comment:
My preference would be to handle this via a rosdep override, if possible.

@wjwwood Is rosdep able to tell apart different versions of OS X the way it can different versions of Ubuntu?